src/GoProControl.cpp

```diff
@@ -1579,9 +1621,10 @@ bool GoProControl::handleHTTPRequest(const char *request)
   return false;
 }
 
-bool GoProControl::sendHTTPRequest(const char *request)
+
+bool GoProControl::sendHTTPRequest(const char *request, uint16_t port)
```

Comment:
make che port a const (constant mean that the value doesn't change inside the function, I have been told that it simplify the work for the compiler)

Reply:
ok